hashintel/hash
# Add loading state to pages list on the sidebar

## 🌟 What is the purpose of this PR?

This PR adds a loading state to the pages list on the sidebar to improve UX & clarity.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1202787635216887/1202889717609170/f) _(internal)_


## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Login to an account with pages
1.  Confirm the loading on the sidebar is visible

## 📹 Demo

https://user-images.githubusercontent.com/39553853/187604404-38667b1e-9164-431a-93b1-53da2a697442.mov
